playground/hmr/invalidation-circular-deps/parent.js

```diff
@@ -0,0 +1,10 @@
+import { value } from './child'
+
+if (import.meta.hot) {
+  import.meta.hot.accept()
```

Comment:
This PR does not work when this `import.meta.hot.accept()` is:
```js
import.meta.hot.accept(() => {
  import.meta.hot.invalidate()
})
```
I guess we need to concatenate the previous chain (that called `import.meta.invalidate`) to the current chain (that happened by the `import.meta.invalidate` call).